scripts/prepare_release.sh

```diff
@@ -72,10 +72,17 @@ function update_changelog() {
     fi
 }
 
+# create the release branch
+git fetch origin/master
+git rebase origin/master
```

Comment:
also if the intent is to ensure that the state of the current branch matches that of origin/master, I'd consider using `git reset --hard origin/master`